content/examples/tooljet/console-compose.yaml.md

```diff
@@ -4,7 +4,7 @@ networks:
   main: {}
 services:
   pomerium:
-    image: pomerium/pomerium:v0.21.1
+    image: cr.pomerium.com/pomerium/pomerium:v0.21.1
```

Comment:
I noticed you updated a `v0.23.0` tag to `latest` above. Should we do the same here, or is there some reason this needs to use an older version?

Reply:
The guide was written to be compatible with v0.21, which requires a [service-level network attribute](https://docs.docker.com/compose/compose-file/05-services/#networks) in the Docker Compose file. We had to update the [v0.21 Quickstart](https://0-21-0.docs.pomerium.com/docs/quickstart) to do this as well. We haven't updated this guide in some time, so I thought it best to keep the current Pomerium version.